src/services/UserService.ts

```diff
@@ -326,6 +326,34 @@ const getUserSecurityGroup = async (userLoginId: string): Promise<any> => {
 
   return userSecurityGroup
 }
+const isUserFulfillmentAdmin = async (userLoginId: string): Promise<any> => {
```

Comment:
The name of the service suggest that boolean value will be returned, but you are returning data. Either return boolean or change the name of the service.